pkg/kine/logstructured/sqllog/otel.go

```diff
@@ -0,0 +1,20 @@
+package sqllog
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/metric"
+)
+
+const name = "sqllog"
+
+var (
+	tracer     = otel.Tracer(name)
```

Comment:
The initialization in this file is mixed (some are in the `init`, some are in the global). Can we uniform that?

Reply:
I'm not sure what you mean here? Could you please explain how this would look like?